common/common.go

```diff
@@ -51,6 +52,11 @@ const (
 	DefaultSystemConfigDir = "/usr/lib/rkt"
 )
 
+// Stage1SharedVolumesDir returns the path where shared (empty) volumes exist on the host.
```

Comment:
This could be a little clearer; "on the host" implies some kind of globality but really it's strictly per-pod